src/core.octotree.js

```diff
@@ -0,0 +1,21 @@
+class OctotreeService {
+  getAccessToken() {
+    return window.store.get(window.STORE.TOKEN);
+  }
+
+  getAccessTokenErrorMessage(settings, jqXHR) {
+    let message = '';
+
+    switch (jqXHR.status) {
```

Comment:
1. Rename to getInvalidTokenMessage.
2. Don't pass jqXHR -> the caller only calls this in 401, so need to check and switch again. Don't pass the entire request settings, just the properties you need.